tools/ipo/macros.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0"?>
+<macros>
+    <xml name="requirements">
+        <requirements>
+            <requirement type="package" version="1.10.0">bioconductor-ipo</requirement>
+            <requirement type="package" version="1.1_5">r-batch</requirement>
+        </requirements>
+    </xml>
+    <xml name="stdio">
+        <stdio>
+            <exit_code range="1" level="fatal" />
+        </stdio>
+    </xml>
+
+    <token name="@COMMAND_SCRIPT@">
+        LANG=C Rscript $__tool_directory__/
+    </token>
+
+    <token name="@COMMAND_LOG_EXIT@">
+        ;
+        return=\$?;
+        mv log.txt $log;
```

Comment:
use && instead of ;  --> use CDATA

Reply:
Hum, I did that on purpose.
I want the log even if the script fails.
But eventually, I need to figure out if we really need the log.

CDATA is included in the main wrapper

Reply:
Done, I removed the log management :)